Automattic/pocket-casts-ios
# CarPlay Star button

Fixes #158 

Adds the Star button to the CarPlay interface. This button is _not_ gated behind the Extra Playback Options toggle since this is not the lock screen or the Car Play Dashboard.

Couldn't get a screen recording from the actual CarPlay unit, but here's one from the simulator:

https://github.com/Automattic/pocket-casts-ios/assets/3250/3365c07f-b0ea-446c-bf10-ff174c512a30

## To test

* Run the app
* Open CarPlay in I/O > External Displays > CarPlay
* Begin playing episode
* Verify Star button is shown at the bottom
* Verify that tapping the Star button correctly marks episode & fills the icon

**NOTE:** When I tested this, the simulator was not really operating the way I expected (weird playback state issues) but everything worked fine in-car. Also, none of these changes really affect any other functionality.

## Checklist
